src/applications/mhv-supply-reordering/pages/nameAndDateOfBirth.js

```diff
@@ -0,0 +1,24 @@
+import {
```

Comment:
Note: Ignore. This file is as generated by the app generator. The form will not work without at least one of these. We will replace this with our own pages.